src/styles/_navbar.scss

```diff
@@ -90,4 +90,50 @@ nav {
       }
     }
   }
+}
+
+/*Responsive CSS rules*/
+@media screen and (max-width: 1200px){
+  nav{
+    .login-text{
+      display: none;
+    }
```

Comment:
You can remove the login-text class to just use `nav button p` instead, but that's up to you